packages/odyssey-react/README.md

```diff
@@ -24,22 +24,6 @@ Install the package:
 $ yarn add @okta/odyssey-react
 ```
 
-Include deprecated global styles with fingerprint extension and fonts:
-
-```html
-<link rel="stylesheet" href="odyssey-deprecated-global.abc123.css" />
-
-<link rel="preconnect" href="https://fonts.gstatic.com" />
-<link
-  rel="stylesheet"
-  href="https://fonts.googleapis.com/css2?family=Public+Sans:ital,wght@0,400;0,600;1,400;1,600&display=swap"
-/>
-<link
-  rel="stylesheet"
-  href="https://fonts.googleapis.com/css2?family=Inconsolata&display=swap"
-/>
-```
-
```

Comment:
we still want to include documentation to describe the fact teams need to include their own fonts - its only the deprecated global stylesheet that is going away

Reply:
This still needs to be updated

Reply:
updated.